README.md

```diff
@@ -2,7 +2,16 @@
 
 # Course Planner
 
-To start your Phoenix app:
+## Development Setup
+
+### Docker
+
+  * Install Elixir dependencies with `docker-compose run web mix deps.get`
+  * Setup your database (create, migrate and seed) with `docker-compose run web mix ecto.setup`
+  * Install Node.js dependencies with `docker-compose run web npm install`
```

Comment:
so, in order to not change too much the project itself I changed the path to mount the project to a different one from the original of the image, then I start the docker-compose with a custom script which creates symbolic links to the download dependencies and compiled files